pdns/rec_channel_rec.cc

```diff
@@ -343,6 +345,33 @@ string doSetCarbonServer(T begin, T end)
 }
 
 template<typename T>
+string doSetDnssecLogBogus(T begin, T end)
+{
+  if (begin == end)
+    return "No DNSSEC Bogus logging setting specified\n";
+
+  if (pdns_iequals(*begin, "on") || pdns_iequals(*begin, "yes")) {
+    if (!arg().mustDo("dnssec-log-bogus")) {
```

Comment:
Because `doSetDnssecLogBogus` does not update `arg`, this code might do the wrong thing when it is called more than once